main.py

```diff
@@ -111,28 +116,17 @@ def main(mainargs=None):
     if args.dump or args.test:
         print("Loading: %s" % (args.input_filename))
         rom = ROMWithTables(args.input_filename)
-        if rom.banks[0][7] == 0x01:
+
+        if args.spoilerformat == "none":
```

Comment:
if console is the default, we can just set the default in the argparser, no need to have a "none" option then.

Reply:
Yeah, this one felt a little weird.  It's there because I felt like --dump should override none - what's the point in dumping the logic if we don't output anything.  The output is still suppressed with 'none' after regular seed generation.

Reply:
Ah, yes, I see now what you mean, then this is fine. I missed the fact that you could do `--dump` on a generated rom, or `--spoilerformat` to get it during the generation.